linux/installer/bin/install-sgx-psw.bin.tmpl

```diff
@@ -30,18 +30,6 @@
 #
 #
 
-modinfo isgx &> /dev/null
-if [[ $? != "0" ]]; then
-    lsmod | grep isgx &> /dev/null
-    if [[ $? = "0" ]]; then
-        echo "Warning: You did not follow the document to install the driver package"
```

Comment:
Based on current code/product status,  the driver dependence check is still valid and it would help SGX newbies, so we decide to keep the check in the PSW installation script.   If the driver name changes in future, only changing the PSW installation script is not enough. We also need to change the loader implementation. 
@jsakkine-intel, could you help to drop this commit? Then we can merge this pull request.